go/genesis/genesis_test.go

```diff
@@ -658,35 +729,48 @@ func TestGenesisSanityCheck(t *testing.T) {
 	// Test staking genesis checks.
 	// NOTE: There doesn't seem to be a way to generate invalid Quantities, so
 	// we're just going to test the code that checks if things add up.
+
+	// NOTE: We will replace the staking part of testDoc with the one from
+	// stakingTests and use stakingTests' entity.
```

Comment:
why this?

Reply:
The `testDoc` used `DebugGenesisState` from `go/staking/tests/debug/debug_stake.go` before:
https://github.com/oasislabs/oasis-core/blob/798fbe38f4665a1cb78cec2d605bf84dcfa51f95/go/genesis/genesis_test.go#L70-L72

Using that for the newly added tests for registry genesis checks was not practical, but I kept them for the staking genesis checks.

I don't have a strong preference though. We could also adopt the staking genesis checks to use the new `testDoc` without replacing its staking part.